openlibrary/plugins/worksearch/schemes/__init__.py

```diff
@@ -14,22 +13,23 @@
 
 
 class SearchScheme:
-    # Set of queries that define the universe of this scheme
-    universe: list[str]
-    # All actual solr fields that can be in a user query
-    all_fields: set[str]
-    # Fields that can be read, but which aren't stored in solr
-    non_solr_fields: set[str]
-    # These fields are fetched for facets and can also be url params
-    facet_fields: set[str]
-    # Mapping of user-only fields to solr fields
-    field_name_map: dict[str, str]
-    # Mapping of user sort to solr sort
-    sorts: dict[str, str | Callable[[], str]]
-    # Default
-    default_fetched_fields: set[str]
-    # Fields that should be rewritten
-    facet_rewrites: dict[tuple[str, str], str | Callable[[], str]]
+    def __init__(self):
+        # Set of queries that define the universe of this scheme
+        self.universe = set()
```

Comment:
Let's bring back the type information here, otherwise it doesn't know what the set contains. 

```suggestion
        self.universe: set[str] = set()
```

And so-on.

Also, let's change `universe` back to a `list` ; we shouldn't need it to be a set.